tests/test_tokenization_base.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+from transformers.models.bert.tokenization_bert import BertTokenizer
```

Comment:
This test should go in an existing test file, for instance the one already testing `BertJapaneseTokenizer` or common tokenizer test file.

Reply:
I'll remove `test_tokenization_base.py`,  and introduce https://github.com/huggingface/transformers/pull/12619#discussion_r668642347 's test instead of this.